tests/packages/sharedlib-in-package/mypkg/__init__.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-FileCopyrightText: 2024 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import sys
+
+
+# start-literalinclude
+def _enable_sharedlib_loading():
```

Comment:
I find the function name a bit misleading. What the function does is to add a path to the list of paths where shared libraries are searched. The capability of loading shared library loading is unaffected. I would name this `_append_to_sharedlib_load_path()` or something like this.

Reply:
SGTM, will change to that

Reply:
Adopted this function name, resolving.